src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -333,3 +333,56 @@
     type: group
     short-summary: Manage resource tags.
 """
+helps['resource link'] = """
+    type: group
+    short-summary: Manage links between resources.
+"""
+helps['resource link create'] = """
+    type: command
+    short-summary: Create a new link between resources.
+    long-summary: Create a new link between resources. A link-id is of the form /subscriptions/{subscription-id}/resourceGroups/{resource-group-name}/{provider-namespace}/{resource-type}/{resource-name}/Microsoft.Resources/links/{link-name}
+    examples:
+        - name: Create a link from link-id to resource-id with
+                notes "some notes to explain this link"
+
+          text: >
+            az resource link create --link-id link-id --target-id resource-id --notes "some notes to explain this link"
```

Comment:
I think it would make the example line super long. What is the right way to approach this?